scripts/minikube.sh

```diff
@@ -18,14 +18,14 @@ function wait_for_ssh() {
 function copy_image_to_cluster() {
     local build_image=$1
     local final_image=$2
-    if [ -z "$(docker images -q "${build_image}")" ]; then
-        docker pull "${build_image}"
+    if [ -z "$(${CONTAINER_CMD} images -q "${build_image}")" ]; then
+        ${CONTAINER_CMD} pull "${build_image}"
     fi
     if [[ "${VM_DRIVER}" == "none" ]]; then
-        docker tag "${build_image}" "${final_image}"
+        ${CONTAINER_CMD} tag "${build_image}" "${final_image}"
         return
     fi
-    docker save "${build_image}" | (eval "$(${minikube} docker-env --shell bash)" && docker load && docker tag "${build_image}" "${final_image}")
+    ${CONTAINER_CMD} save "${build_image}" | (eval "$(${minikube} docker-env --shell bash)" && docker load && docker tag "${build_image}" "${final_image}")
```

Comment:
is podman available inside minikube vm? for podman load and podman tag?

Reply:
Podman exists inside minikube, and I didn't see any doc that says load and tag this not supported.

However, when I try to run load and tag exporting the podman-env though minikube, it doesn't work.

When I ssh into the vm and list images
```
[0] pkalever 😎 ceph-csi✨ minikube ssh
                         _             _            
            _         _ ( )           ( )           
  ___ ___  (_)  ___  (_)| |/')  _   _ | |_      __  
/' _ ` _ `\| |/' _ `\| || , <  ( ) ( )| '_`\  /'__`\
| ( ) ( ) || || ( ) || || |\`\ | (_) || |_) )(  ___/
(_) (_) (_)(_)(_) (_)(_)(_) (_)`\___/'(_,__/'`\____)

$ whoami 
docker
$ podman images
ERRO[0000] cannot find mappings for user docker: open /etc/subuid: no such file or directory 
REPOSITORY   TAG   IMAGE ID   CREATED   SIZE

```

I will investigate further and keep this work open for future PRs.

In this PR the CONTAINER_CMD setting is only for local, and for now let the minikube use docker only.